lib/record_store/record/caa.rb

```diff
@@ -0,0 +1,46 @@
+module RecordStore
+  class Record::CAA < Record
+    attr_accessor :flags, :tag, :value
+
+    LABEL_REGEX = '[a-z0-9](?:-*[a-z0-9])*'
+    DOMAIN_REGEX = /\A#{LABEL_REGEX}(?:\.#{LABEL_REGEX})\z/
```

Comment:
How does this differ from FQDN from the parent class? I guess this is needed to handle `mailto:` values.

Reply:
see my reply to angelo's review re: RFC and domain value... the URLs are not handled with this regex and are only validated for `iodef` tags